### ISSUE TITLE
Install role dependencies with ansible-galaxy?

### ISSUE THREAD

**Issue by bborysenko** [metacloud/molecule#71]
Hi,

Could you please give piece of advice about the right way to test ansible role that has some dependencies with molecule? For the moment I use a bit different  directory layout and override some molecule defaults:

```
$ tree .
...
├── tests
│   ├── playbook.yml       <-- playbook to execute during `molecule converge`
│   ├── requirements.yml   <-- requirements for `ansible-galaxy`
│   ├── roles              <-- extra roles installed with `ansible-galaxy`
│   │   └── yum-epel
│   │   └── ...
│   ├── serverspec         <-- `serverspec_dir`
│   │   └── ...
...

$ cat molecule.yml
---
molecule:
  serverspec_dir: tests/serverspec
  testinfra_dir: tests/testinfra
  ansible_config_template: /Users/bbo/.config/molecule/ansible.cfg.j2
ansible:
    playbook: tests/playbook.yml
...

$ cat /Users/bbo/.config/molecule/ansible.cfg.j2
[defaults]

roles_path = tests/roles:../
library    = library
```

And before `molecule converge` I manually run:

```
ansible-galaxy install -r tests/requirements.yml -p tests/roles
```

But I guess that it should be some better way to do it. It will be great to have configuration option with path to a file with requirements for `ansible-galaxy` and install them automatically before `molecule converge`.

**Comment by retr0h** [comment]
@bborysenko I've not given this piece too much thought.  My initial expectation would have people configure roles_path and library to their liking, pretty much like you have done.  This is how we do it as well. I'm kinda curious to see how people want to tackle dependencies, and go from there.  Cool?

**Comment by abrown-sg** [comment]
Fixed by #105.
